src/pages/error/404.ts

```diff
@@ -0,0 +1,10 @@
+import { renderDOM } from "../../utils/renderDOM";
+import { ErrorBlock } from "../../components/error-block/errorBlock";
+import './../index.css'
```

Comment:
Стоит придерживаться одного вида кавычек на расширение файлов
```suggestion
import { renderDOM } from './../utils/renderDOM';
import { ErrorBlock } from '../../components/error-block/errorBlock';
import './../index.css'

```